andela/devrepublic-bn-backend
# #170947562 Users get notifications for approvals

#### What does this PR do?
add a `Users get notifications for approved requests` feature

#### Description of Task to be completed?
- added a table for notification
- installed socket.io
- test the feature
- documented the feature
- internationalized all messages

#### How should this be manually tested?
- clone this repo
- run npm install to install all dependencies
- run the server (npm run reset:dev)
- make a put request to `/api/v1/trips/{requestId}/confirm`
- you will get an email with a link to the approved request

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#170947562](https://www.pivotaltracker.com/story/show/170947562)

#### Screenshots (if appropriate)
<img width="1123" alt="Screen Shot 2020-03-11 at 12 06 12" src="https://user-images.githubusercontent.com/41383503/76405477-d9e00880-6390-11ea-84c9-b9f5bf913eed.png">


#### Questions:
- N/A
